checkout/access-checkout-android/src/test/java/com/worldpay/access/checkout/views/PANLengthFilterTest.kt

```diff
@@ -214,30 +214,30 @@ class PANLengthFilterTest {
         assertNull(filteredString)
     }
 
-    @Test
-    fun givenAnIdentifiedCardWithMultipleValidationRulesAndInputDoesNotExceedMaximumThenShouldNotFilter() {
-        val fourteenDigitCardInput = "40000000000000"
-
-        val cardValidationRule1 = CardValidationRule(null, null, 16, null)
-        val cardValidationRule2 = CardValidationRule(null, null, 15, null)
-        val cardBrand = CardBrand("visa", emptyList(), null, listOf(cardValidationRule1, cardValidationRule2))
-        val validationResult = ValidationResult(partial = true, complete = false)
-        given(cardValidator.validatePAN(fourteenDigitCardInput)).willReturn(Pair(validationResult, cardBrand))
-
-        // Add a '0' to existing string
-        val filteredString = panLengthFilter.filter("0", 0, 1, SpannableStringBuilder(fourteenDigitCardInput), fourteenDigitCardInput.length, fourteenDigitCardInput.length)
-
-        // Verify that no filtering has been done
-        assertNull(filteredString)
-    }
+//    @Test
```

Comment:
should this still be commented out?

Reply:
This test is no longer relevant as we no longer have brands with multiple pan rules